openapi/openapi-points.ytt.yaml

```diff
@@ -334,3 +334,37 @@ paths:
           schema:
             $ref: "#/components/schemas/WriteOrdering"
       responses: #@ response(reference("UpdateResult"))
+  /collections/{collection_name}/points/batch:
+    put:
```

Comment:
I wonder if POST may be more appropriate here. Though I know the issue description mentioned to use PUT. What do you think?

Reply:
As `points/delete` is a POST, and therefore does not signal idempotency, having an RPC-like endpoint that includes that operation should probably also be a POST, like you suggest.

Reply:
I changed it to POST